decentraland/unity-explorer
# fix: fixes on backpack to avoid issues with quick page / category changes

## What does this PR change?

When clicking too fast on the page numbers under the grid on the wearable screen, it was possible to cause a crash on the backpack. 
This also fixes issues causes when changing a wearable category and then, before it finishes loading, changing the pages quickly. It would soon go to non-existent pages and break.

## How to test the changes?

Just go to the backpack/wearables section and try switching pages furiously, even clicking the same page several times. It should not crash and should load the correct wearables.
Particularly the last page, try clicking on it many many many times. Nothing should break :)
